admin-guides/sprint-calls-prep.md

```diff
@@ -1,26 +1,39 @@
-# Prep for Sprint Calls
+# Prep for Weekly Calls
 
-## Who does this Prep
+## Who is Responsible
 
-Ideally, the lead of each call, or a specific person.
+- The host stays the same for each month.
+- On the last week of each month, identify a host for the following month.
+- If the host can’t make it some week, they are responsible for delegating to someone and making sure they have permissions at least a day ahead.
 
-## Steps to Take
-
-**Advanced preparation**
+## Host Responsibilities
 
 On Wednesday of the preceding week:
 - Make sure that [node-github-issue-bot](https://github.com/ipfs/node-github-issue-bot) has created the next sprint issue. It should use templates from [here](https://github.com/ipfs/node-github-issue-bot/tree/master/templates). If it breaks, look at [the README in ipfs/infrastructure](https://github.com/ipfs/infrastructure/tree/master/sprintbot).
 - Manually fill out the Notes link with [the templates](https://github.com/ipfs/pm/tree/master/templates). To not do this manually, follow [this issue](https://github.com/ipfs/node-github-issue-bot/issues/8).
 
-By Friday of the preceding week:
-- Adjust the calendar times to match the Sprint issue. To not manually do this, follow [this issue](https://github.com/ipfs/node-github-issue-bot/issues/4).
+#### Before the Call
+
+1. Make sure agenda is posted beforehand in GitHub using the current format (see #647 for an example). Ideally, this should be a day or more early so people can add items before the meeting. (Also consider proposed changes in #636.)
+1. Post a notice just prior to and at the meeting time on IRC (#ipfs & #ipfs-dev), Twitter (@ipfsbot), & relevant channels.
```

Comment:
I think we should be specific here so the host knows what to do. 😄  Are there other relevant channels? (I don’t know them if there are.) We should list them or remove “& relevant channels.”